src/test/java/com/webank/webase/front/util/SampleKeys.java

```diff
@@ -0,0 +1,30 @@
+package com.webank.webase.front.util;
+
+import java.math.BigInteger;
+
+import org.fisco.bcos.web3j.crypto.Credentials;
+import org.fisco.bcos.web3j.crypto.ECKeyPair;
+import org.fisco.bcos.web3j.utils.Numeric;
+
+/** Keys generated for unit testing purposes. */
+public class SampleKeys {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'SampleKeys' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/gh/WeBankFinTech/WeBASE-Front/pullRequest?prid=4823531)